src/libcaf-gfortran-descriptor.h

```diff
@@ -51,7 +51,7 @@ typedef struct gfc_descriptor_t {
   void *base_addr;
   size_t offset;
   ptrdiff_t dtype;
-#if (__GNUC__ >= 8)
+#ifdef GCC_GE_8
   ptrdiff_t span;
```

Comment:
I think this answers my earlier question.  Apparently, the only change with GCC>=8 is internal: a new descriptor member `span` (added during Paul's work on PDT support if I recall correctly).  I don't imagine therein any reason to be mention this change in user documentation.

Reply:
Well, none that I know of. Most of the development in gcc-8 went past me. Besides two small bugfixes there is nothing that jumps into my mind.